ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -59,10 +61,22 @@ class BaseScreenComponent extends React.PureComponent<Props, State> {
   }
 
   private showHelp = () => {
+    // tslint:disable-next-line:no-unused-expression
+    this.props.dark &&
+      setStatusBarColorAndBackground(
+        "dark-content",
+        customVariables.colorWhite
+      );
     this.setState({ isHelpVisible: true });
   };
 
   private hideHelp = () => {
+    // tslint:disable-next-line:no-unused-expression
```

Comment:
@CrisTofani 
This code lgtm
Anyway if you want to get the same logic but in a functional way you could do:

```typescript
maybeDark(this.props.dark).map(_ => {
      setStatusBarColorAndBackground(
        "dark-content",
        customVariables.colorWhite
      );
    });
```

**maybeDark** is a function (see below) that creates an [Option](https://gcanti.github.io/fp-ts/modules/Option.ts.html). An option is a type can hold 2 value: some or none.
If I call map on it, it means I want execute the provided function only when Option is some

```typescript
const maybeDark = fromPredicate(
  (isDark: boolean | undefined = undefined) => isDark === true
);
```

The code is pretty the same but it is more speaking: "it could be dark, if yes a function will be executed"
If you prefer your code, no problem, I will merge anyway ;)

Reply:
I want to come closer to the Functional approach! I'll catch your suggestion and try to change my code!

Any further suggestions as this one is really appreciated!